cel2sql.go

```diff
@@ -922,6 +920,14 @@ func IsListType(typ *exprpb.Type) bool {
 	return ok
 }
 
+func IsStringType(typ *exprpb.Type) bool {
```

Comment:
Google SQL also supports nullable arrays:
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#array_nulls
Should we support them as well? I guess it requires similar changes in `IsListType` function.